src/SME.SGP.WebClient/src/paginas/Gestao/AtribuicaoEsporadica/componentes/DreDropDown.js

```diff
@@ -55,7 +55,10 @@ function DreDropDown({ form, onChange, label }) {
       containerVinculoId="containerFiltro"
       valueText="desc"
       placeholder="Diretoria Regional De Educação (DRE)"
-      disabled={listaDres.length === 1}
+      disabled={
+        listaDres.length === 1 ||
+        (typeof desabilitado === 'boolean' && desabilitado === true)
```

Comment:
Comparação de desabilitado com true não é necessária, pode ser só (typeof desabilitado === 'boolean' && desabilitado), se não for boolean nem passa na primeira validação nem que tenha conteúdo